### PR TITLE
Kentucky general election dates

### PR DESCRIPTION
**Pull request by askpatrickw**
Kentucky's official state motto is "United we stand, divided we fall".

### CODE REVIEW

File: states/kentucky/elections.toml [hunk 1]
```diff
@@ -22,3 +22,28 @@ in_person_by = 2020-06-23
...
+
+[20201103.poll.early]
+in_person_start = 2020-10-13
+in_person_by = 2020-11-02
+sources = ["https://vrsws.sos.ky.gov/abrweb/"]
+
+[20201103.poll]
+in_person_start = 2020-11-03T06:00:00
+in_person_by = 2020-11-03T18:00:00
+received_by = 2020-11-03T18:00:00
```
Comment (tannewt) [inline states/kentucky/elections.toml]: What does received_by mean here?

File: states/kentucky/elections.toml [hunk 2]
```diff
@@ -22,3 +22,28 @@ in_person_by = 2020-06-23
...
 original_date = 2020-11-03
+
+[20201103.registration]
+online_by = 2020-10-05T16:00:00
+received_by = 2020-10-05T16:00:00
+in_person_by = 2020-10-05T16:00:00
+sources = ["https://elect.ky.gov/Resources/Documents/2020%20Election%20Calendar%20with%20Primary%20Postponement.pdf"]
+
+[20201103.absentee.application]
+received_by = 2020-10-27
```
Comment (tannewt) [inline states/kentucky/elections.toml]: The popup on https://vrsws.sos.ky.gov/abrweb/ says online_by 10/9 midnight.

**Review by tannewt — COMMENTED** [review]
A couple questions.

File: states/kentucky/elections.toml (hunk 2 above)
Comment (askpatrickw) [reply states/kentucky/elections.toml]: PDF says:

>Last day to apply for mail-in absentee ballot (not later than close of business   7   days   before   election).   Applications,   including   the   declaration accompanying a federal write-in absentee ballot, must be received by this day.

KY's site is a mess. I guess we should error to the earlier date. Agree?

File: states/kentucky/elections.toml (hunk 1 above)
Comment (askpatrickw) [reply states/kentucky/elections.toml]: Its a mistake. Good catch. The PDF has received by block on 11/3 but that is for absentee.

File: states/kentucky/elections.toml (hunk 2 above)
Comment (tannewt) [reply states/kentucky/elections.toml]: Ya, I think the earlier is better. Otherwise we risk folks not getting their ballots.

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**